ci/mill-bootstrap.patch

```diff
@@ -1,30 +1,30 @@
 diff --git a/.github/workflows/autofix.yml b/.github/workflows/autofix.yml
-index ce356a24ebc..ab1cfeb53d2 100644
+index fff08f9d91e..4b869becf2c 100644
 --- a/.github/workflows/autofix.yml
 +++ b/.github/workflows/autofix.yml
-@@ -8,7 +8,6 @@ jobs:
-       - uses: actions/checkout@v4
-
+@@ -10,7 +10,6 @@ jobs:
+       - uses: sbt/setup-sbt@v1
+ 
```

Comment:
This looks suspicious. It's shouldn't be necessary to change the CI setup as part of switching the Mill version.